crates/starknet-server/tests/test_dump_and_load.rs

```diff
@@ -217,4 +220,54 @@ mod dump_and_load_tests {
 
         remove_file(dump_file_name);
     }
+
+    #[tokio::test]
+    async fn dump_load_endpoint() {
+        let dump_file_name = "dump_endpoint";
+        let devnet_dump = BackgroundDevnet::spawn_with_additional_args(&[
+            "--dump-path",
+            dump_file_name,
+            "--dump-on",
+            "exit",
```

Comment:
Why is it started with `--dump-on`?

Reply:
right, it can just be `BackgroundDevnet::spawn()`

Reply:
Fixed in 5218df73f399e95a2cf17d76c6f6ce3ca7003fe6